include/solvers/vof.h

```diff
@@ -163,6 +170,11 @@ class VOF : public AuxiliaryPhysics<dim, TrilinosWrappers::MPI::Vector>
    * physics. It does not concern the output of the solution using the
    * DataOutObject, which is accomplished through the attach_solution_to_output
    * function
+   *
+   * @param scratch_data The scratch data which is used to store
+   * the calculated finite element information at the gauss point.
+   * See the documentation for VOFScratchData for more
+   * information
```

Comment:
I think this block is not necessary as a header here no?
```suggestion
   *
   * @param scratch_data The scratch data which is used to store
   * the calculated finite element information at the gauss point.
   * See the documentation for VOFScratchData for more
   * information
```

Reply:
Maybe, I just corrected a typo here!